pulsebot/pulse_dispatch.py

```diff
@@ -277,7 +277,9 @@ def comment():
                             }
                         # TODO: reopen closed bugs on backout
                         if ('leave-open' not in values.get('keywords', {}) and
-                                not is_backout and not info.leave_open):
+                                not is_backout and not info.leave_open and
+                                values.get('status', '') not in
+                                ('VERIFIED', 'CLOSED')):
```

Comment:
Why not add RESOLVED?

Reply:
I considered it at the time but was being minimalist in my changes, but I'll add it in.